suites/pacific/rbd/tier-2_rbd_mirror_image_operations.yaml

```diff
@@ -169,7 +169,12 @@ tests:
       clusters:
         ceph-rbd1:
           config:
-            imagesize: 2G
+            rep_pool_config:
```

Comment:
@manasagowri initially it has to enable `journal`  based then snapshot based, It use to work earlier runs please refer https://access.redhat.com/documentation/en-us/red_hat_ceph_storage/5/html-single/block_device_guide/index#converting-journal-based-mirroring-to-snapshot-based-mirroring_block

```2023-10-06 20:14:24,275 (cephci.test_rbd_mirror_journal_to_snap) [INFO] - cephci.Regression.rbd.13.cephci.tests.rbd_mirror.test_rbd_mirror_journal_to_snap.py:26 - As Expected writing data from secondary got failed, mirrored image got locked from primary in journal based mirroring
2023-10-06 20:14:24,275 (cephci.test_rbd_mirror_journal_to_snap) [INFO] - cephci.Regression.rbd.13.cephci.ceph.ceph.py:1558 - Running command rbd mirror image disable rep_pool_AKRaxAmkHm/rep_image_vpfhCxFgMd --cluster ceph on 10.0.195.161 timeout 600
2023-10-06 20:14:24,845 (cephci.test_rbd_mirror_journal_to_snap) [ERROR] - cephci.Regression.rbd.13.cephci.ceph.ceph.py:1594 - Error 22 during cmd, timeout 600
2023-10-06 20:14:24,845 (cephci.test_rbd_mirror_journal_to_snap) [ERROR] - cephci.Regression.rbd.13.cephci.ceph.ceph.py:1595 - 2023-10-06T10:44:24.696-0400 7ff09b2b63c0 -1 librbd::api::Mirror: image_disable: cannot disable mirroring in the current pool mirroring mode```
It should work as per this doc, Is it product issue?

Reply:
Got it during review meeting.